src/modules/stores/authentication/authentication.models.ts

```diff
@@ -19,8 +19,16 @@ export class AuthenticationModel {
     id: string,
     email: string,
     displayName: string,
-    type: '' | UserTypeEnum,
-    roles: UserRoleEnum[],
+    roles: {
```

Comment:
Didn't notice on the backend, but do we really need the created/updated/deleted at/bys ... even the active is doubtful for me. Other opinions